rust/perspective-viewer/src/rust/config/expressions.rs

```diff
@@ -13,110 +13,84 @@
 use std::borrow::Cow;
 use std::collections::HashMap;
 
-use itertools::Itertools;
 use serde::{Deserialize, Serialize};
 
-#[derive(Serialize, Deserialize, Clone, PartialEq, Debug)]
+#[derive(Deserialize, Clone, PartialEq, Debug)]
 #[serde(untagged)]
-pub enum Expressions {
+pub enum ExpressionsDeserde {
```

Comment:
What makes this type unique that it requires the qualifier `Deserde` in the name (as opposed to the other components of `ViewerConfig`)? There is no other `Expressions` type, so the fact that it only supports deserialization doesn't disambiguate it.